Engines/Wine/Verbs/FAudio/script.js

```diff
@@ -26,29 +28,19 @@ Wine.prototype.faudio = function (faudioVersion) {
         .extract();
 
     var forEach = Array.prototype.forEach;
-    var sys32dir = this.system32directory();
+    var sys64dir = this.system64directory();
     var faudioDir = this.prefixDirectory() + "/FAudio/faudio-" + faudioVersion;
     var self = this;
 
-    forEach.call(ls(faudioDir + "/x32"), function (file) {
+    forEach.call(ls(faudioDir + "/x64"), function (file) {
```

Comment:
Please use an arrow function lambda expression. This allows you to directly use `this` inside the passed function:

```javascript
ls(faudioDir + "/x64").forEach(file => {
   ...
}); 
```

If you get an error because of the forEach please try:

```javascript
ls(faudioDir + "/x64").foreach(file => {
   ...
}); 
```

and

```javascript
Java.from(ls(faudioDir + "/x64")).forEach(file => {
   ...
}); 
```

Reply:
I'm not sure. That part of the script works.